packages/hooks/use-draggable/index.ts

```diff
@@ -55,13 +57,17 @@ export const useDraggable = (
     const { offsetX, offsetY } = transform
 
     const onMousemove = (e: MouseEvent) => {
+      if (!isDragging.value) {
+        isDragging.value = true
+      }
```

Comment:
We can enable the dragging state into `onMouseDown` for a better perf & instant style.

```suggestion
const { offsetX, offsetY } = transform
isDragging.value = true

const onMousemove = (e: MouseEvent) => {
```

Reply:
I ran a [repl](https://play.vuejs.org/#eNp9Uk1P6zAQ/CsrXyhSX/v0HqeQIr56gAMg4OiLSTapqWNb9qYNqvrfWbtQioQ4WLJnZ9czo92IC+8nqx5FIcpYBe0JIlLvz6TVnXeBYAMBmzGsFVWLedNgRbCFJrgOjrjvSFppK2cjgY7XQbWtti3MUs+oUSbicSIcNI9GxzA7g420cNAxWSnTY8KUwUAj7tryKac7TayGH4SdN4qQXwBlrVeQLgB1GqJeDM6koMBjxEch0lsGX1S1bIPrbV2AC8q2eApe1TV/XMC/v344hbWuaVFAp4Y/7IbQ0n7KeZofSQXiUd9M7j5LrCwJgNQSocs+yikLZLicHsgWY0GR5ze6nbxGZzn1HIQUleu8Zuv3njSnKUWxiyjVlDFufZux9OP4E68WWC1/wF/jkDApHgJGDCvWuK+xjRbZWyrPn+5w4Pu+2Lm6N8z+pfiI0Zk+adzRLjlUln3Ay2pv8u5wSs9xPnCa8dNUjoyZ28yXglfo6hfrX3L/T05yHy+G2L4DDhvnQA==) to see when dragstart is triggered and it is effectively triggered on mouse move.

We can stay with the current change :+1: .